fileHub/js-app/app/services/state-management/mutator/mutator.js

```diff
@@ -67,5 +68,18 @@ export const mutator = (mutatorName, details, state) => {
         isCurrentUserInfoFetching: false,
         fetchingCurrentUserDetailsErrorMessage: details.error,
       });
+    case DELETE_ITEM_MUTATOR.FETCHING_STARTED:
+      return Object.assign({}, state, {
+        removingFile: details.removingFile,
+      });
+    case DELETE_ITEM_MUTATOR.FETCHING_COMPLETED:
+      return Object.assign({}, state, {
+        removingFile: null,
+      });
+    case DELETE_ITEM_MUTATOR.FETCHING_FAILED:
+      return Object.assign({}, state, {
+        removingFile: null,
```

Comment:
This should close modal in current implementation. Please check why it is not closed.